ci-operator/step-registry/distributed-tracing/tests/tracing-ui/integration/distributed-tracing-tests-tracing-ui-integration-commands.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# List of variables to check.
+vars=(
+  CYPRESS_SKIP_COO_INSTALL
+  CYPRESS_COO_UI_INSTALL
+  CYPRESS_KONFLUX_COO_BUNDLE_IMAGE
+  CYPRESS_CUSTOM_COO_BUNDLE_IMAGE
+  CYPRESS_DT_CONSOLE_IMAGE
+)
+
+# Loop through each variable.
+for var in "${vars[@]}"; do
+  if [[ -z "${!var}" ]]; then
+    unset "$var"
+    echo "Unset variable: $var"
+  else
+    echo "$var is set to '${!var}'"
+  fi
+done
+
+# Set proxy vars.
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+  source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+## skip all tests when console is not installed.
+if ! (oc get clusteroperator console --kubeconfig=${KUBECONFIG}) ; then
+  echo "console is not installed, skipping all console tests."
+  exit 0
+fi
+
+# Function to copy artifacts to the artifact directory after test run.
+function copyArtifacts {
+  if [ -d "gui_test_screenshots" ]; then
+    cp -r gui_test_screenshots "${ARTIFACT_DIR}/gui_test_screenshots"
```

Comment:
Not sure if better change /gui_test_screenshots to /coo_tracing_gui_test_screenshots

Reply:
Its a standard name across all the ui tests for the artifacts directory used by our Red Hat teams.